File: Data Structures & Algorithms/intersection-of-two-arrays/submission-0.cpp
class Solution {
public:
    vector<int> intersection(vector<int>& nums1, vector<int>& nums2) {
        unordered_map<int,int> m;
        vector<int> res;
        for(int num:nums1){
            m[num] =1;
        }
        for(int num:nums2){
            if(m[num]==1){
                res.push_back(num);
                m[num] = 0;
            }
        }
        return res;
    }
};

File: Data Structures & Algorithms/pow-x-n/submission-0.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        res=1.0
        p = abs(n)
        while p:
            if p%2==1:
                res*=x
            x*=x
            p//=2
        return res if n>=0 else 1/res